packages/manager/modules/anthos/src/dashboard/host/set-state/controller.js

```diff
@@ -0,0 +1,31 @@
+export default class {
+  /* @ngInject */
+  constructor($translate, AnthosTenantsService) {
+    this.$translate = $translate;
+    this.AnthosTenantsService = AnthosTenantsService;
+  }
+
+  $onInit() {
+    this.isStateful = this.host.stateful;
+  }
+
+  setState() {
+    this.AnthosTenantsService.setHostState(
+      this.serviceName,
+      this.host.id,
+      this.isStateful,
+    )
+      .then((hostData) => {
+        this.host.update(hostData);
+        this.goBack();
```

Comment:
success message not required?

Reply:
A success message is not required here, since the state of the host is directly updated